Detections/SecurityEvent/SolorigateNamedPipe.yaml

```diff
@@ -59,24 +59,30 @@ query: |
   | extend Account =  strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
   )
   )
-  | extend timestamp = TimeGenerated, AccountCustomEntity = Account, HostCustomEntity = Computer
-
+  | extend timestamp = TimeGenerated
+  | extend HostName = tostring(split(Computer, ".")[0]), DomainIndex = toint(indexof(Computer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(Computer, DomainIndex + 1), Computer)
+  | extend AccountName = tostring(split(Account, "@")[0]), AccountUPNSuffix = tostring(split(Account, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
```

Comment:
Account is actually a Name + NTDomain in this case.  So you can map it, but you have to split it as follows:

| extend AccountName = tostring(split(Account, "\\")[1]), AccountNTDomain = tostring(split(Account, "\\")[0])